orchagent/vxlanorch.cpp

```diff
@@ -626,26 +626,20 @@ bool VxlanTunnelOrch::addOperation(const Request& request)
     SWSS_LOG_ENTER();
 
     auto src_ip = request.getAttrIP("src_ip");
-    if (!src_ip.isV4())
-    {
-        SWSS_LOG_ERROR("Wrong format of the attribute: 'src_ip'. Currently only IPv4 address is supported");
-        return true;
-    }
 
     IpAddress dst_ip;
     auto attr_names = request.getAttrFieldNames();
     if (attr_names.count("dst_ip") == 0)
     {
-        dst_ip = IpAddress("0.0.0.0");
+        if(src_ip.isV4()) {
+            dst_ip = IpAddress("0.0.0.0");
+        } else {
+            dst_ip = IpAddress("::");
+        }
     }
     else
     {
         dst_ip = request.getAttrIP("dst_ip");
-        if (!dst_ip.isV4())
```

Comment:
Need to verify that stc_ip family matches dst_ip family, and also matches endpoint_ip family for every tunnel next hop

Reply:
@marian-pritsak , i'm not sure if we need the endpoint_ip family check. It would be slightly complex and may not be a valid use-case. src_ip and dst_ip check can be added.

Reply:
I have pushed the family check for src_ip and dst_ip. @marian-pritsak, can this be approved if it looks good